api/Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN apt-get update \
 COPY pyproject.toml poetry.lock ./
 RUN poetry install --sync --no-cache --no-root
 
+# install jsonpath-ng==1.6.1
+RUN poetry add jsonpath-ng==1.6.1
```

Comment:
hello, pls add your dependencies at `pyproject.toml` instead of Dockerfile

Reply:
Thank you for your suggestion. I have moved the dependencies from the Dockerfile to the pyproject.toml file as recommended.